app/src/main/res/layout-sw600dp-port/profile_chooser_profile_view.xml

```diff
@@ -59,38 +59,29 @@
 
         <TextView
           android:id="@+id/profile_name_text"
-          android:layout_width="wrap_content"
-          android:layout_height="wrap_content"
           android:ellipsize="end"
-          android:fontFamily="sans-serif-medium"
+          style="@style/Heading2"
           android:gravity="center"
           android:maxLines="2"
           android:singleLine="false"
           android:text="@{viewModel.profile.name}"
-          android:textColor="@color/white"
-          android:textSize="20sp" />
+          android:textColor="@color/white" />
 
         <TextView
           android:id="@+id/profile_last_visited"
-          android:layout_width="wrap_content"
-          android:layout_height="wrap_content"
-          android:fontFamily="sans-serif-light"
+          style="@style/Subtitle1"
           android:textColor="@color/white"
-          android:textSize="16sp"
           android:layout_marginTop="4dp"
           android:textStyle="italic"
           android:visibility="@{viewModel.profile.lastLoggedInTimestampMs > 0 ? View.VISIBLE : View.GONE}"
           profile:lastVisited="@{viewModel.profile.lastLoggedInTimestampMs}" />
 
         <TextView
           android:id="@+id/profile_is_admin_text"
-          android:layout_width="wrap_content"
-          android:layout_height="wrap_content"
-          android:fontFamily="sans-serif-light"
+          style="@style/Subtitle1"
```

Comment:
And here.

Reply:
Done

Reply:
@BenHenning Please has this been resolved?